src/Test.UnitTests.Sarif.Driver/TestRuleBehaviors.cs

```diff
@@ -9,6 +9,24 @@ namespace Microsoft.CodeAnalysis.Sarif.Driver
     internal enum TestRuleBehaviors
     {
         None = 0,
-        LogError = 0x1
+
+        // Produce an error for every scan target
```

Comment:
>t [](start = 48, length = 1)

Periods at ends of sentences.

Reply:
Thanks for your heart that is true.

---
In reply to: [322894005](https://github.com/microsoft/sarif-sdk/pull/1670#discussion_r322894005) [](ancestors = 322894005)